libraries/shared.rb

```diff
@@ -0,0 +1,28 @@
+#
+# Author:: Brent Montague (<bmontague@cvent.com>)
+# Cookbook Name:: octopus-deploy
+# Library:: shared
+#
+# Copyright:: Copyright (c) 2015 Cvent, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module OctopusDeploy
+  # A container to hold the shared logic instead of attributes
+  module Shared
+    def fancy_bool(bool)
+      bool.to_s.capitalize
```

Comment:
@jonathanmorley its not needed as in I could just do the bool but I want to follow the .net standard